helper/resource/importstate/import_block_with_id_test.go

```diff
@@ -82,7 +82,15 @@ func TestImportBlock_WithID_ExpectError(t *testing.T) {
 				resource "examplecloud_container" "test" {
 					location = "eastus"
 					name     = "somevalue"
-				}`,
+				}
+
+				import {
```

Comment:
This test change is a consequence of #484.

Curiously, the tests did not run in https://github.com/hashicorp/terraform-plugin-testing/pull/484 💭